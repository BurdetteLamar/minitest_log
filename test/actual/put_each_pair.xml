<log>
  <summary verdicts='0' failures='0' errors='0'/>
  <each_pair name='put_each_pair' class='Hash'>
    <![CDATA[
a => 0
b => 1
]]>
  </each_pair>
</log>
