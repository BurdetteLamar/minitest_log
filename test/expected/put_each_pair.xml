<log>
  <data_ name='put_each_pair' class='Hash' method=':each_pair' size='2'>
    <![CDATA[
a => 0
b => 1
]]>
  </data_>
</log>
